op-service/signer/cli.go

```diff
@@ -14,21 +14,23 @@ const (
 	AddressFlagName  = "signer.address"
 )
 
-func CLIFlags(envPrefix string) []cli.Flag {
+func CLIFlags(envPrefix string, category string) []cli.Flag {
```

Comment:
This `category` is added in order to render the signer-related flags under the `5. PEER-TO-PEER` section when `--help` is ran.